src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,46 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SEPARATOR = ",";
+    private static final String RESULT_HEADER = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
https://github.com/mate-academy/jv-work-with-file/blob/master/checklist.md#divide-program-logic

Please split method logic to separate private methods (read file content, form report from file content and write report to file) and use these methods in the public one